edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/EdgeHubCertificates.cs

```diff
@@ -14,6 +14,14 @@ namespace Microsoft.Azure.Devices.Edge.Hub.Service
 
     public class EdgeHubCertificates
     {
+        enum OperatingMode
```

Comment:
I think this enum is not necessary because you have the switch just after you set the mode. You could move the code in directly in if-else.

Reply:
I had it that way but I thought this is better for readability. I can revert if needed.

Reply:
Removed the enum and addressed the comment.